docs/source/migrations/1.0.mdx

```diff
@@ -6,69 +6,69 @@ description: From 0.x to 1.0
 import SetupCodegen from "../../shared/setup-codegen/combined.mdx"
 import SeparateLocalCacheMutationsNote from "../../shared/separate-local-cache-mutation-note.mdx"
 
-Apollo iOS 1.0 provides a stable API that uses modern Swift language conventions and features. Among other improvements, it features:
+Apollo iOS 1.0 provides a stable API that uses modern Swift language conventions and features.
 
-- New generated models that improve readability and functionality while reducing generated code size in most cases
-- New code generation tooling that is written in pure Swift code
-- Support for using generated models in multi-module projects
-- Type-safe APIs for cache key resolution
+Among other improvements, Apollo iOS 1.0 features new:
 
-Because most of the library's core concepts are unchanged, the migration to 1.0 from the legacy 0.x versions of Apollo iOS should not require serious rewrites in most cases. This page describes the most important changes, along with how to migrate an existing project from Apollo iOS 0.x to 1.0.
+- Code generation tooling written in pure Swift code.
+- Generated models that improve readability and functionality while reducing generated code size.
+- Support for using generated models in multi-module projects.
+- Type-safe APIs for cache key resolution.
 
-> **To start your migration to Apollo iOS 1.0:**
-> - You should first read about the [key changes](#key-changes) highlighted below.
-> - Then, follow the [step-by-step instructions](#step-by-step-instructions).
-> - Lastly, use the [breaking changes](#breaking-changes) section to help you resolve any remaining build errors in your code.
+This article describes the significant changes for this major version of Apollo iOS and a walk-through of migrating an existing project from Apollo iOS 0.x to 1.0.
+
+> **Steps to migrate to Apollo iOS 1.0:**
+> 1. Begin by reading about the [key changes](#key-changes).
+> 2. Next, follow our [step-by-step instructions](#step-by-step-instructions) to upgrade your existing app.
```

Comment:
I don't know why we wanted to change the heading from "Step-by-step instructions" to "Migration walk-through", but we need to then change this link also.

I personally prefer step-by-step instructions, but am fine with either, as long as we fix the link!